tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -4,13 +4,13 @@ const httpHelper = require('../helpers/httpHelper')
 const userSettings = require('../helpers/userSettings')
 const fetch = require('node-fetch')
 const assert = require('assert')
+const { join } = require('../helpers/path')
 
 When('user {string} is sent a notification', function (user) {
   const body = new URLSearchParams()
   body.append('user', user)
 
-  return fetch(
-    client.globals.backend_url + '/ocs/v2.php/apps/testing/api/v1/notifications',
+  return fetch(join(client.globals.backend_url, '/ocs/v2.php/apps/testing/api/v1/notifications'),
```

Comment:
Better to move the url into a separate variable

Reply:
fixed